superset-frontend/src/components/TelemetryPixel/index.tsx

```diff
@@ -47,7 +47,7 @@ const TelemetryPixel = ({
   const pixelPath = `https://apachesuperset.gateway.scarf.sh/pixel/${PIXEL_ID}/${version}/${sha}/${build}`;
   return process.env.SCARF_ANALYTICS === 'false' ? null : (
     <img
-      referrerPolicy="no-referrer-when-downgrade"
+      referrerPolicy="no-referrer"
```

Comment:
Is this change related to the pr or just a rebase mistake?

Reply:
It was a rebase mistake! Reverted!